content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+aliases: ["/news/contributor-interview-snor42"]
+title: "Freelance developper and Translator: meet SNor42"
```

Comment:
```suggestion
title: "Freelance Developper and Translator: Meet SNor42"
```

Reply:
I think it should be a "Developer", right?